torch/csrc/jit/interpreter.cpp

```diff
@@ -228,8 +228,9 @@ struct CanEmitInline {
     scanBlock(graph->block());
   }
   bool canInline(Value* v) {
-    return v->node()->kind() != prim::Param && v->uses().size() == 1 &&
-        v->node()->outputs().size() == 1;
+    return v->node()->kind() != prim::Param &&
+           v->node()->kind() != prim::BailOut && v->uses().size() == 1 &&
```

Comment:
can you add a comment why bailouts cannot be emitted inline?

Reply:
yes, I definitely hit an issue if we let `BailOut`s to be emitted inline.